src/model/application.model.ts

```diff
@@ -38,7 +39,7 @@ export interface ApplicationData {
     is_secure_register?: string;
     who_is_registering?: string;
     update?: updateType.Update;
-    is_listed_as_property_owner?: string;
+    remove?: removeType.Remove;
```

Comment:
Probably don't need to do this right now (and perhaps after [this](https://github.com/companieshouse/overseas-entities-api/pull/369) PR has been merged) but need to remember to persist the new `remove` block data in Mongo at some point. So changes in the SDK and API to support this.

I don't think the `is_listed_as_property_owner` field would be saved in Mongo anyway (as a filter page response) but we need to do this for the statement field.